client/ayon_hiero/api/plugin.py

```diff
@@ -675,15 +713,12 @@ def __init__(self, cls, track_item, **kwargs):
         self.track_index = int(track_item.parent().trackIndex())
 
         # adding tag.family into tag
-        if kwargs.get("avalon"):
-            self.tag_data.update(kwargs["avalon"])
+        if avalon:
```

Comment:
Avalon is the predecessor of Pype 2.0, before OpenPype, before AYON. **What** is still Avalon in AYON here? And do we really still need that - it looks like something we should be burning to the ground and get rid of? Or put a big comment above over why this is _still_ needed.

Reply:
Avoid using "Avalon" in the code. Rename it to "AYON" if it makes more sense. For example, change the `avalon` attribute to `instance_data` or just `data`.


However, since the data is added only to pass `tag.family` (productType), which becomes `instance.data["productType"]`, we should consider revamping the code entirely, possibly in a future PR. For now, a simple argument name change should suffice.